### PR TITLE
feat: Disable fragment context menu item in CPE for controls with no stable id

### PR DESCRIPTION
**Pull request by nikmace**
Feat for #1904.

- Adds check for RTA context menu to disable Add Fragment menu item for controls with unstable ids.

### CODE REVIEW

**Review by GDamyanov — APPROVED** [review]
Small and simple change.
Code looks good.
Excellent code coverage.
Manual testing not done.

File: packages/preview-middleware-client/test/unit/adp/init-dialogs.test.ts [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 import RuntimeAuthoringMock from 'mock/sap/ui/rta/RuntimeAuthoring';
 
-import { DialogNames, handler, initDialogs, isControllerExtensionEnabled } from '../../../src/adp/init-dialogs';
+import { DialogNames, handler, initDialogs, isFragmentCommandEnabled, isControllerExtensionEnabled } from '../../../src/adp/init-dialogs';
 import AddFragment from '../../../src/adp/controllers/AddFragment.controller';
 import ControllerExtension from '../../../src/adp/controllers/ControllerExtension.controller';
 import ExtensionPoint from '../../../src/adp/controllers/ExtensionPoint.controller';
 import ElementOverlay from 'sap/ui/dt/ElementOverlay';
-import FlUtils from 'sap/ui/fl/Utils';
+
```
Comment (voicis) [inline packages/preview-middleware-client/test/unit/adp/init-dialogs.test.ts:18]: This is what probably causing lint errors.

Comment (nikmace) [reply packages/preview-middleware-client/test/unit/adp/init-dialogs.test.ts]: Hopefully that fixes it: [24b72be](https://github.com/SAP/open-ux-tools/pull/1905/commits/24b72bef8a7f14575a6b9aee8252c5d0c7aac7d6)

**Review by voicis — APPROVED** [review]
- Changeset present
- Simple code changes
- Great test coverage

**Review by GDamyanov — APPROVED** [review]
- Simple change.
- Code looks good.
- Great test coverage.
- Did not tested manually.

**Pull request merged**